filmfest/urls.py

```diff
@@ -6,12 +6,14 @@
 from wagtail.wagtailadmin import urls as wagtailadmin_urls
 from wagtail.wagtailcore import urls as wagtail_urls
 
+import search.views
 
 urlpatterns = [
     url(r'^django-admin/', include(admin.site.urls)),
     url(r'^admin/', include(wagtailadmin_urls)),
-] + i18n_patterns('',
-    url(r'^search/$', 'search.views.search', name='search'),  # noqa
+] + i18n_patterns(
+    '',
+    url(r'^search/$', search.views.search, name='search'),
```

Comment:
i'd rather keep `# noqa` in order to ignore PEP8 rules in cases like this in future